decimal64.go

```diff
@@ -5,13 +5,17 @@ import (
 	"math/bits"
 )
 
-// Decimal64 represents an IEEE 754 64-bit floating point decimal number.
-// It uses the binary representation method.
-type Decimal64 struct {
-	bits uint64
-}
-
 type flavor int
+type roundingMode int
+type discardedDigit int
+
+const (
+	unset discardedDigit = iota
```

Comment:
`unset` is too generic. Need to related the name to `discardedDigits` somehow. I'd be happy with something verbose like `unsetDiscardedDigit`, since it's never actually used. A perfectly cromulent alternative is to start with `eq0 discardedDigit = 1 + iota`.

Reply:
I'm just going to get rid of unset
as you said, it doesn't actually get used apart from initialisation.